rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -85,6 +85,13 @@ def ensure_global(name, default):
         # TIP: if using the sphinx-bootstrap-theme, you need
         # "treeViewIsBootstrap": True,
         "exhaleExecutesDoxygen": False,
+        # Maps markdown files to the "md" lexer, and not the "markdown" lexer
+        # Pygments registers "md" as a valid markdown lexer, and not "markdown"
+        "lexerMapping": {{r".*\.md": "md",}},
+        # This mapping will work when `exhale` supports `:doxygenpage:` directives
+        # Check `svenevs/exhale#111`
+        "customSpecificationsMapping": utils.makeCustomSpecificationsMapping(
+        lambda kind: [":project:", ":path:", ":content-only:"] if kind == "page" else []),
```

Comment:
For reviewers, the [`customSpecificationsMapping`]() argument allows users to:

> For a given input ``kind``, return the list of reStructuredText specifications for the associated Breathe directive.

Essentially, this mapping should allow `:doxygenpage:` inputs to only show the markdown text (`:content-only:`), rather than redirecting the user to a separate html page for the markdown text.